src/Block/FormatterBlockService.php

```diff
@@ -65,12 +77,12 @@ public function configureSettings(OptionsResolver $resolver): void
         ]);
     }
 
-    public function getBlockMetadata($code = null): Metadata
```

Comment:
I guess there should be an upgrade note with these BC breaks.

Reply:
Not sure if this is a BC break, because both block version will work

Reply:
I meant because of removing public methods.

Reply:
Reverted it

Reply:
If it's deprecated, it should trigger a deprecation.

Reply:
And what about a `NEXT_MAJOR` comment to remove it.

Reply:
> And what about a `NEXT_MAJOR` comment to remove it.

Adding a `NEXT_MAJOR` comment is redundant for a deprecated method

Reply:
Decision in dev-kit was to use both https://github.com/sonata-project/dev-kit/issues/206

As an example, when I tried to remove some deprecation on SonataAdmin@master, I was only looking for `NEXT_MAJOR` in my IDE.